### ISSUE TITLE
shortcut modifier to zoom pauses while zooming in/out

### ISSUE THREAD

**Issue by chadrolfs** [adobe-photoshop/spaces-design#1845]
open a file, select any tool *except* the Select tool, hold down cmd/ctrl + space + click and drag right and left

Result: zoom in/out will stutter/pause before it becomes smooth.

**Comment by volfied** [comment]
Dupe of https://github.com/adobe-photoshop/spaces-design/issues/1844 ?

**Comment by volfied** [comment]
Re-opening this. @chadrolfs I've reverted my change and scrubby zoom is back.

Please refer to this techdoc from 10 years ago for the possible fixes.

If it's still performance wise bad, please let us know and Jesper will look into it.

http://www.adobe.com/support/techdocs/331723.html

**Comment by iwehrman** [comment]
1. That tech doc is crazy. 
2. I'm not sure how it addresses this issue.

I still think we should disable this zoom shortcut. Why did you revert the change?

**Comment by chadrolfs** [comment]
occurs on 10.10, works great on 10.11

**Comment by iwehrman** [comment]
@chadrolfs Does this happen for you in 10.10 in standard PS as well? I'm wondering if we need to keep the github issue open. Anyway, it works great for me here, but I'm on 10.11.

**Comment by chadrolfs** [comment]
No, it doesn't happen in standard PS on 10.10 but does in DS only.
